packages/components/psammead-navigation/src/index.jsx

```diff
@@ -170,19 +171,14 @@ export const NavigationLi = ({
         script={script}
         currentLink="true"
         service={service}
-        data-navigation={`${link}_${url}`}
```

Comment:
This IMO makes this a breaking change, up to you wether you follow it, as it’s very minor

Reply:
Done. Bumped the versions